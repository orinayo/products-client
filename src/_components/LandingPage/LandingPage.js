import React from 'react'

const LandingPage = () => {
  return (
    <main className='landing-page'>
      <h1 className='heading-primary u-font-cursive'>
        In with the NEW
      </h1>
    </main>
  )
}

export default LandingPage
