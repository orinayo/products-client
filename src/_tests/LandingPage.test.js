import { shallow } from 'enzyme'
import React from 'react'
import LandingPage from '../_components/LandingPage/LandingPage'

it('expect to render LandingPage component', () => {
  expect(shallow(<LandingPage />)).toMatchSnapshot()
})