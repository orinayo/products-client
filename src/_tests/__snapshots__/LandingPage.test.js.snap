// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render LandingPage component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1
        className="heading-primary u-font-cursive"
      >
        In with the NEW
      </h1>,
      "className": "landing-page",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "In with the NEW",
        "className": "heading-primary u-font-cursive",
      },
      "ref": null,
      "rendered": "In with the NEW",
      "type": "h1",
    },
    "type": "main",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1
          className="heading-primary u-font-cursive"
        >
          In with the NEW
        </h1>,
        "className": "landing-page",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "In with the NEW",
          "className": "heading-primary u-font-cursive",
        },
        "ref": null,
        "rendered": "In with the NEW",
        "type": "h1",
      },
      "type": "main",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
